### PR TITLE
Enabled three_step_cwl centaur test.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: centaur/src/main/resources/standardTestCases/three_step.test [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 workflowType: CWL
 workflowTypeVersion: v1.0
 backends: [Local]
-backendsMode: "only"
 tags: [localdockertest]
 
 files {
   wdl: three_step/three_step.cwl
   inputs: three_step/inputs.json
+  options: three_step/three_step.options
```
Comment (cjllanwarne) [inline centaur/src/main/resources/standardTestCases/three_step.test]: Does this really fix the problem or just force the JES centaur to run the test on its (should be removed) “Local” backend?

File: centaur/src/main/resources/standardTestCases/three_step.test [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 }
 
-metadata { }
+metadata {
+  "submittedFiles.workflowType": CWL
+  "submittedFiles.workflowTypeVersion": v1.0
+  "calls.wc.executionStatus": Done
+  "calls.ps.executionStatus": Done
+  "calls.cgrep.executionStatus": Done
```
Comment (cjllanwarne) [inline centaur/src/main/resources/standardTestCases/three_step.test]: Can we test outputs too? To make sure CWL support isn’t regressing

File: centaur/src/main/resources/standardTestCases/three_step.test (hunk 1 above)
Comment (kshakir) [reply centaur/src/main/resources/standardTestCases/three_step.test]: Specifically, this forces TES-centaur to run this test on the (should be removed, maybe, see below) "Local" backend.

JES-centaur passes [`-elocaldockertest`](https://github.com/broadinstitute/cromwell/blob/c6ed64617c51c572863b87d324fa8e68fa085b1a/src/bin/travis/resources/centaur.wdl#L34), meaning that _all_ tests tagged as such are skipped. Meanwhile, TES uses a very specific [blacklist](https://github.com/broadinstitute/cromwell/blob/c6ed64617c51c572863b87d324fa8e68fa085b1a/src/bin/travis/testCentaurTes.sh#L71-L78) of incompatible tests, allowing those that can run to specify via a workflow-options file that the test should run on the (should be removed..?) Local.

File: centaur/src/main/resources/standardTestCases/three_step.test (hunk 2 above)
Comment (kshakir) [reply centaur/src/main/resources/standardTestCases/three_step.test]: Added a new issue https://github.com/broadinstitute/cromwell/issues/2924 that also describes the current issue with 3step.

**Comment by kshakir** [comment]
FYI was continuation of #2835

**Pull request merged**